### DISCUSSION TITLE
Testing Speed in a Multi-WAN Environment

### DISCUSSION THREAD

**Discussion by mircsicz** [alexjustesen/speedtest-tracker#1520] [Ideas]
Hi Guys,

first of all thanks for this awesome project!

Here's my scenario:
I've two locations with Dual-WAN. What I'ld like to do is be able to run Speedtest-Tracker on both WAN's. So I could track the speed of my Fiber and my Starlink.

I imagine assigning a fixed test-Server to WAN A & B and then set the default GW in my Firewall to go through a specific WAN for that Speedtest Target-Host...

What do you think?

**Comment by KossikCZ** [comment]
I have a dual-WAN environment at home too. The way I dealt with it was a creation of secondary container with a separate IP, which is then redirected to 2nd WAN for this traffic.

**Comment by mircsicz** [comment]
Thanks, didn't even consider that...
